Dockerfile

```diff
@@ -34,3 +34,15 @@ EXPOSE 8000
 RUN pip install gunicorn
 ENTRYPOINT ["/bin/bash", "-c", "gunicorn  --forwarded-allow-ips='*' --chdir '/app/todo_app/' -b 0.0.0.0 'app:create_app()'"]
 
```

Comment:
Sorry something I missed when I reviewed module 5 that needs to be fixed:

- You have `COPY . /code` and do not have a .dockerignore file to stop the .env file being copied in. So if you build an image locally it contains your secrets. We don't want that.